### PR TITLE
fix: login redirect error

### LINKED ISSUE #88: fix: url after failed login should be /login NOT /login/authenticate

### PR DESCRIPTION
**Pull request by JackieSL1**
# Description

Replaces the `login/authenticate` endpoint with a POST to `login`. This fixes the redirection to `login/authenticate`.

## Issues

- closes #88 

# How to test

Manual test:
1. Go to `/login`
2. Enter an invalid username and password
3. It should take you to `/login`, and you should see an error message

Integration tests:
* Run `mvn test`

# Checklist
**I've added tests**
- [ ] Yes
- [x] No - Updated existing tests

**I've updated the DB schema and UML diagram**
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by JackieSL1** [comment]
@BraedenKloke Not sure if this is exactly what you wanted from a fix, but it works and I think having GETs and POSTs to the same endpoint is standard

**Review by ajbalara — APPROVED** [review]
Looks good to me, but maybe want to wait for @BraedenKloke's review before merging?

**Review by BraedenKloke — APPROVED** [review]
Looks great!

Yes, this is the correct implementation. Having the same endpoint accept both GET and POST requests.

**Pull request merged**
